docs/reference/hooks.md

```diff
@@ -458,13 +458,13 @@ def construct_forms_for_user(user, queryset):
 
 ## Editor interface
 
-Hooks for customising the editing interface for pages and snippets.
+Hooks for customizing the editing interface for pages and snippets.
 
 (register_rich_text_features)=
 
 ### `register_rich_text_features`
 
-Rich text fields in Wagtail work with a list of 'feature' identifiers that determine which editing controls are available in the editor, and which elements are allowed in the output; for example, a rich text field defined as `RichTextField(features=['h2', 'h3', 'bold', 'italic', 'link'])` would allow headings, bold / italic formatting and links, but not (for example) bullet lists or images. The `register_rich_text_features` hook allows new feature identifiers to be defined - see [](rich_text_features) for details.
+Rich text fields in Wagtail work with a list of 'feature' identifiers that determine which editing controls are available in the editor, and which elements are allowed in the output; for example, a rich text field defined as `RichTextField(features=['h2', 'h3', 'bold', 'italic', 'link'])` would allow headings, bold/italic formatting, and links, but not (for example) bullet lists or images. The `register_rich_text_features` hook allows new feature identifiers to be defined - see [](rich_text_features) for details.
```

Comment:
I prefer spaces between slashes, it wraps nicer, but not going to push either way unless there are others who think we should leave this.